app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -57,6 +160,17 @@ export class KubernetesV2RunJobStageConfig extends React.Component<IStageConfigP
         />
         <h4>Manifest Configuration</h4>
         <YamlEditor value={this.state.rawManifest} onChange={this.handleRawManifestChange} />
+        <h4>Output</h4>
+        <StageConfigField label="Capture Output From" helpKey="kubernetes.runJob.captureSource">
+          <div>
+            <select className="form-control input-sm" value={stage.consumeArtifactSource} onChange={this.sourceChanged}>
```

Comment:
just for consistency with the other React selects, would you mind using the `Select` component from `react-select`?

Reply:
can do!